hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OFSPath.java

```diff
@@ -44,7 +48,7 @@
    * /vol1/buc2/dir3/key4  vol1           buc2           (empty)      dir3/key4
    * /vol1/buc2            vol1           buc2           (empty)      (empty)
    * /vol1                 vol1           (empty)        (empty)      (empty)
-   * /tmp/dir3/key4        tempVolume     tempBucket     tmp          dir3/key4
+   * /tmp/dir3/key4        tmp            <username>     tmp          dir3/key4
```

Comment:
How do we allow userA to access userB's tmp bucket? This may be needed for the ETL workload where an injest process run as user A and the process process run as user B.

Reply:
As of now userA can't access userB's temp with ofs:// . Because OFSPath parses `ofs://tmp/dir1/dir2` into `volume=tmp bucketname=<USER_TEMP_BUCKET_NAME>, key(in bucket)=dir1/dir2`

Reply:
I'm thinking of adding some `.reserved` like HDFS did to allow user to bypass the mount or add some options, if there is a need.

Anyway this may not fit in this jira.

Reply:
agree, can you add the proposal to the design doc in the umbrella JIRA?